test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * 
+ */
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
```

Comment:
in general, import only what is necessary. can use Eclipse's 'organise imports' to do this automatically